kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -37,6 +41,8 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
   val credentialsManager = new HadoopCredentialsManager()
   // this lazy is must be specified since the conf is null when the class initialization
   lazy val sessionConfAdvisor: SessionConfAdvisor = PluginLoader.loadSessionConfAdvisor(conf)
+  lazy val staticBatchSecretId: UUID = conf.get(BATCH_STATIC_SECRET_ID)
```

Comment:
If we want to share across servers later, we can create a string literally

Reply:
yes, we can define a static UUID string for multiple kyuubi server instances.

Reply:
I mean remove the configuration

Reply:
Just define a random UUID? 

Such as `c2ee5b97-3ea0-41fc-ac16-9bd708ed8f38` ?

Reply:
yes